src/modules/previewpane/SvgPreviewHandler/SvgPreviewControl.cs

```diff
@@ -55,6 +55,7 @@ public override void DoPreview<T>(T dataSource)
                     }
                 }
 
+                svgData = SvgPreviewHandlerHelper.ScaleSvg(svgData);
```

Comment:
I was wondering if `ScaleSvg` error handing should be more granular, so that if it fails, we still show the preview unscaled. What your thoughts?

Reply:
Yeah, sure I didn't think about it. I would move it to a separate try-catch expression. So if an error occurs it will just display it unscaled. As this is not totally breaking preview I will not call `PreviewError()` which is used in other exceptions I will just call logger for the exception.